manifests/t/Tyrrrz/DiscordChatExporter/CLI/2.42.6/Tyrrrz.DiscordChatExporter.CLI.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Tyrrrz.DiscordChatExporter.CLI
+PackageVersion: 2.42.6
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
+    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack.4.5

```

Comment:
Alright thanks, I'll remove the VCRedist then. I added the .NET Framework 4.x because of this weird prompt appearing, I'll see which ones I need after some tests.

Reply:
Can confirm now that VCRedist is not needed explicitly. I got this info:

![image](https://github.com/microsoft/winget-pkgs/assets/81419612/173a55d5-39e4-45d6-af8c-960986084ff0)

and installed the .NET 8 runtime and it's working now.